packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -72,8 +77,8 @@ const Checkbox = React.forwardRef(
                         'dc-checkbox__box--disabled': disabled,
                         'dc-checkbox--grey-disabled': disabled && greyDisabled,
                     })}
-                    {...(withTabIndex?.length > 0 ? { tabIndex: withTabIndex } : {})}
-                    tabIndex='0'
+                    {...(withTabIndex && withTabIndex?.length > 0 ? { tabIndex: withTabIndex } : {})}
+                    tabIndex={0}
```

Comment:
```suggestion
                    tabIndex={withTabIndex}
```
@iman-fs It seems to be the same thing 🤔 Change `withTabIndex` type to `number` and give it a default value of `0` 🤔 Would you check please 🙏🏻

Reply:
I could find the prop `withTabIndex` used only in `packages/account/src/Components/personal-details/personal-details.jsx`, so as @farzin-fs suggested u can change it to number in this component as well as in the place it is used. Or accept string and convert it to number while using it in Checkbox component

Reply:
@farzin-fs @likhith-deriv Thank you guys. I changed my code. good catch Farzin :+1: